fieldservice_stock/models/fsm_equipment.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2018 - TODAY, Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class FSMEquipment(models.Model):
+    _inherit = 'fsm.equipment'
+    _description = "FSM Equipment Inherits"
+
+    lot_id = fields.Many2one(
+        'stock.production.lot', string='Lot/Serial #')
```

Comment:
@smangukiya We also need a `product_id` field to filter the lot_id. Both fields should be required.